.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: go
+
+go: 1.4
+
+script:
+    - git config --global user.email "you@example.com"
+    - git config --global user.name "Your Name"
+    - test -z "$(go fmt)"
+    - go vet
+    - go test
+    - go test -race
```

Comment:
Let's add golint in a separate commit, along with fixing the dozen or so things it complains about.